### PR TITLE
Feature/improve experiment message template handling

### PR DESCRIPTION
**Pull request by timohuber**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,4 +33,4 @@
     "htmx.org": "^1.6.0",
     "tippy.js": "^6.3.7"
   }
-}
+}
```
Comment (mabiede) [inline package.json:36]: please remove this change

File: pool/web/handler/admin_message_templates.ml [hunk 2]
```diff
@@ -112,6 +118,144 @@ let update req =
...
+      |> CCOption.to_result Pool_common.Message.(NotFound Field.MessageTemplate)
+      |> Lwt_result.lift
+    in
+    let form_context =
+      if CCOption.is_some current_template
+      then `Update template
+      else `Create template
+    in
+    let flash_fetcher = CCFun.const None in
+    (* TODO: flash_fetcher *)
```
Comment (mabiede) [inline pool/web/handler/admin_message_templates.ml:245]: ping todo

File: resources/index.scss [hunk 3]
```diff
@@ -469,7 +469,14 @@ table.simple {
...
 
-// Width base classes
+// Display EMail Template rich text
+.force-normalize-fonts {
+    h1, h2, h3, h4, h5 {
+        font-size: $body-font-size;
+    }
+}
+
+// Width base classes (TODO: Framework integration)
```
Comment (mabiede) [inline resources/index.scss:479]: ping todo

Comment (timohuber) [reply resources/index.scss:479]: Will remove the comment when I update the framework package. I am not sure yet what we will need to include in the framework

**Review by mabiede — APPROVED** [review]

**Pull request merged**